code/package.json

```diff
@@ -23,8 +23,15 @@
     "url": "https://github.com/atulmy/crate/issues"
   },
   "homepage": "https://github.com/atulmy/crate#readme",
-  "dependencies": {},
+  "dependencies": {
```

Comment:
This was added in when I installed knex, we should no longer need this.

Reply:
Is this comment regarding line 29?

Reply:
Regarding the whole block.